apps/home/templates/apps_home/blocks/teaser_columns.html

```diff
@@ -30,10 +30,10 @@ <h3 class="title">{{ value.title }}</h3>
                 <p class="text">
                     {{ value.body }}
                     {% if value.button.internal_link %}
-                        <a href="{{ value.internal_link.url }}" class="more" data-mainlink="true" title="">{{ value.button.link_text }}</a>
+                        <a href="{{ value.internal_link.url }}" class="more ms-1" data-mainlink="true" title="">{{ value.button.link_text }}</a>
                     {% endif %}
                     {% if value.button.external_link %}
-                        <a href="{{ value.external_link.url }}" class="extern" data-mainlink="true" title="">{{ value.button.link_text }}</a>
+                        <a href="{{ value.external_link.url }}" class="extern ms-1" data-mainlink="true" title="">{{ value.button.link_text }}</a>
```

Comment:
Should we maybe open external links on a new page? Just a thought and should probably be answered by @hmkay or @maxliqd (Please tag me if you answer this, otherwise only @phillimorland will get notified.)